conf/messages

```diff
@@ -1130,13 +1130,13 @@ businessmatching.registerservices.other.title = What other service do you need t
 businessmatching.registerservices.details = More information about these categories
 
 businessmatching.registerservices.existing = You’ve told us your business already does:
-businessmatching.registerservices.servicename.lbl.01 = Accountancy services
-businessmatching.registerservices.servicename.lbl.02 = Bill payment services
-businessmatching.registerservices.servicename.lbl.03 = Estate agency services
-businessmatching.registerservices.servicename.lbl.04 = High value dealing
-businessmatching.registerservices.servicename.lbl.05 = Money service business activities
-businessmatching.registerservices.servicename.lbl.06 = Trust or company services
-businessmatching.registerservices.servicename.lbl.07 = Telephone, digital and IT payment services
+businessmatching.registerservices.servicename.lbl.01 = Accountancy service provider
+businessmatching.registerservices.servicename.lbl.02 = Bill payment service provider
+businessmatching.registerservices.servicename.lbl.03 = Estate agency business
+businessmatching.registerservices.servicename.lbl.04 = High value dealer
+businessmatching.registerservices.servicename.lbl.05 = Money service business
```

Comment:
So these labels have been changed within messages.

The change was specifically for the page "How much total net profit did your business make in the last 12 months, from the following activities?"

These labels are used a number of times throughout the service for instance
 AMLSTurnoverController
 businessMatching models etc.

Does the removal of "activities" apply to all of these places. Because as I see it the ticket was to change this in one place.

Reply:
its been discussed and agreed with Lisa and Jeoff that the above are the set that we will use through out the service.
if the impact is huge on the acceptance test, lets re-assess the impact

Reply:
There are 10 acceptance test failures due to this so ill get to work on them now.